sensu-agent/plan.sh

```diff
@@ -0,0 +1,30 @@
+pkg_name=sensu-agent
+pkg_filename=sensu-agent
+pkg_origin=core
+pkg_version="2.0.0-beta.2-4"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
```

Comment:
I'd like to double check this.  I can't find reference on their official site and I'm not sure if I'm looking at the correct repo, but the senso-go repository is MIT licensed. 

https://github.com/sensu/sensu-go/blob/master/LICENSE

Reply:
@smacfarlane That's the correct repo. Both the agent and backend are inside the repo. I'm going to update both to be MIT licensed.